weed/s3api/filer_multipart.go

```diff
@@ -23,7 +23,7 @@ type InitiateMultipartUploadResult struct {
 }
 
 func (s3a *S3ApiServer) createMultipartUpload(ctx context.Context, input *s3.CreateMultipartUploadInput) (output *InitiateMultipartUploadResult, code ErrorCode) {
-	uploadId, _ := uuid.NewV4()
+	uploadId := uuid.NewV4()
```

Comment:
Can we upgrade the uuid package version instead?

Reply:
Sure, updated the patch